process/incubation.md

```diff
@@ -119,40 +140,33 @@ Changes to the Swift Server Ecosystem index page will be announced by the SSWG u
 * Ecosystem
   * Uses SwiftPM
   * Integrated with critical SSWG ecosystem building blocks, e.g., Logging and Metrics APIs, SwiftNIO for IO
-* Concurrency / IO
-  * Packages should be non-blocking (w/ async API) unless not possible (blocking C libs, etc)
-  * There should be as little (preferably no) wrapping of NIO as possible. Exposing NIO types directly will go a long way for making packages compatible.
-  * Blocking code should be wrapped in NonBlockingIOPool (like Vapor's SQLite package)
-* Testing, CI and Release
-  * Have unit tests for Linux
-  * CI setup, including testing PRs and the main branch
-  * Follow semantic versioning
-* Conventions and Style
-  * Error handling
-  * Follow Swift Code style guidelines
-  * Uses force unwraps and force tries only as preconditions, ie. conditions that the programmer regards as impossible or programmer error. All force tries/unwraps should come with a comment stating the reasons
-  * Prefer native Swift over C wrapping, where appropriate
-  * Does not use `*Unsafe*` unless interfacing with C
-    * Exceptions to uses of `*Unsafe*` constructs are acceptable when appropriately documented why they are absolutely necessary.
-    * When `*Unsafe*` is used in this manner, its is expected to be accompanied by enhancement request tickets for the root cause in Swift, SwiftNIO, or a different offending library.
 * Longevity
-  * Must be a team of 2+ developers
   * Must be from a team that has more than one public repository (or similar indication of experience)
   * SSWG should have access / authorization to graduated repositories in case of emergency
-  * Adopt the [SSWG Security Best Practices](#security-best-practices))  
+  * Adopt the [SSWG Security Best Practices](#security-best-practices))    
+* Testing, CI and Release
+  * Have unit tests for Linux
+  * CI setup, including testing PRs and the main branch
+  * Follow semantic versioning, with at least one published pre-release (e.g. 0.1.0, 1.0.0-beta.1) or release (e.g. 1.0.0)
 * Licensing
-  * Apache 2
+  * Apache 2, MIT or BSD (Apache 2 recommended)
+* Conventions and Style
+  * Adopt [Swift API Design Guidelines](https://swift.org/documentation/api-design-guidelines/)
+  * Follow [SSWG Technical Best Practices](#technical-best-practices)) when applicable.
+  * Prefer to adopt code formatting tools and integrate them into the CI
 
 ## Graduation Requirements
 
 * [Minimal Requirements](#minimal-requirements)
+* Have stable API (no pending/planned breaking API changes), with at least one published major release (e.g. 1.0.0)
 * Support new GA versions of Swift within 30d
-* CI setup for two latest Swift.org recommnded versions of Swift
-* CI setup for two latest versions of Swift.org recommnded Linux distributions
+* CI setup for two latest Swift.org recommended versions of Swift
+* CI setup for at least one of Swift.org recommended Linux distributions
+* CI setup for each platform supported by the library or tool
```

Comment:
Swift-NIO runs on anything from Apple Watch up - it doesn't have CI for all the device platforms.  

As the swift server working group are we only interested in potential server platforms eg macOS, linux, windows?